matRadGUI.m

```diff
@@ -1044,7 +1044,7 @@ function UpdatePlot(handles)
     end
 end
 
-if get(handles.radioBtnIsoCenter,'Value') == 1 && get(handles.popupTypeOfPlot,'Value') == 1 && ~isempty(pln)
+if get(handles.radioBtnIsoCenter,'Value') == 1 && get(handles.popupTypeOfPlot,'Value') == 1 && ~isempty(pln) && size(pln.isoCenter,1) == 1
     hIsoCenterCross = matRad_plotIsoCenterMarker(handles.axesFig,pln,ct,plane,slice);
```

Comment:
also the GUI elment for the isocenter should also work with pln.isoCenter being an array of multiple isoCenter. I think we should show 'multiple isocenters' if there are really different isocenter for different beams but if there is only one joint isocenter we should display the same isocenter - also if someone want to modify the isocenter we should allow that and then copy the isocenter to all beams?!